src/XrmDefinitelyTyped/Resources/Extensions/xrm_ext_9-.d.ts

```diff
@@ -1227,7 +1227,7 @@ declare namespace Xrm {
     }
 
     interface AddNotificationObject {
-        actions?: actionsObject | null;
+        actions?: actionsObject[] | null;
```

Comment:
Is the null needed? The argument is already optional. The same applies to the actionsObject typing. I know that is how it was, but we might aswell fix it now :)

Reply:
Checked and there is no need to specify nulls for optionals in typescript. Also done smoke checking that actual addNotification method doesn't throw when actions or messages are not specified.